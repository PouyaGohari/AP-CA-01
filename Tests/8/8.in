3
pa1 pa2 pa3
pa1:0 YES|1 NO|2 IF_NECESSARY
pa2:0 YES|2 NO|1 YES
pa3:1 NO