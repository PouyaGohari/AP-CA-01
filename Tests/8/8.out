Time slot 0:
YES: pa1 pa2 
NO: 
IF_NECESSARY: pa3 
###
Time slot 2:
YES: 
NO: pa2 
IF_NECESSARY: pa1 pa3 
###
Time slot 1:
YES: pa2 
NO: pa1 pa3 
IF_NECESSARY: 
###
